winpr/libwinpr/utils/wlog/wlog.c

```diff
@@ -231,7 +231,7 @@ BOOL WLog_WritePacket(wLog* log, wLogMessage* message)
 
 BOOL WLog_PrintMessageVA(wLog* log, wLogMessage* message, va_list args)
 {
-	BOOL status;
+	BOOL status = FALSE;
```

Comment:
Looks good, anyway (not your code) shouldn't we print a message when the kind of message is not known ?